packages/yarnpkg-fslib/sources/ZipFS.ts

```diff
@@ -439,42 +439,34 @@ export class ZipFS extends BasePortableFakeFS {
     if (p === null)
       throw new Error(`Unimplemented`);
 
-    let fd = this.openSync(p, `r`);
-
-    const closeStream = () => {
-      if (fd === -1)
-        return;
-      this.closeSync(fd);
-      fd = -1;
-    };
-
-    const stream = Object.assign(new PassThrough(), {
-      bytesRead: 0,
-      path: p,
-      close: () => {
-        clearImmediate(immediate);
-        closeStream();
-      },
-      _destroy: (error: Error | undefined, callback: (error?: Error) => void) => {
-        clearImmediate(immediate);
-        closeStream();
-        callback(error);
-      },
-    });
+    const fd = this.openSync(p, `r`);
+
+    const stream = Object.assign(
+      new PassThrough({
+        emitClose: true,
+        autoDestroy: true,
```

Comment:
Aren't both of those already [set to `true` by default](https://nodejs.org/api/stream.html#stream_new_stream_writable_options)?

Reply:
Depends on the node version, check the history for the docs you linked to